.azure-pipelines-templates/install_deb.yml

```diff
@@ -31,15 +34,38 @@ steps:
     workingDirectory: build
     displayName: Recovery benchmark for installed CCF
 
-  - script: |
-      set -ex
-      unsafe_build=$(grep unsafe /tmp/install_prefix) || true
-      if [[ -n "$unsafe_build" ]]; then
-        ./tests/test_install_build.sh -DUSE_UNSAFE_VERSION=ON
-      else
-        ./tests/test_install_build.sh
-      fi
-    displayName: Test building a sample against installed CCF
+  - ${{ if or(eq(parameters.target, 'SGX'), eq(parameters.target, 'SGXIceLake')) }}:
+      - script: |
+          set -ex
+          unsafe_build=$(grep unsafe /tmp/install_prefix) || true
+          if [[ -n "$unsafe_build" ]]; then
+            ./tests/test_install_build.sh -DUSE_UNSAFE_VERSION=ON -DCOMPILE_TARGET=sgx
+          else
+            ./tests/test_install_build.sh -DCOMPILE_TARGET=sgx
+          fi
+        displayName: Test building a sample against installed CCF (SGX)
+
+  - ${{ if eq(parameters.target, 'Virtual') }}:
+      - script: |
+          set -ex
+          unsafe_build=$(grep unsafe /tmp/install_prefix) || true
+          if [[ -n "$unsafe_build" ]]; then
+            ./tests/test_install_build.sh -DUSE_UNSAFE_VERSION=ON -DCOMPILE_TARGET=virtual
+          else
+            ./tests/test_install_build.sh -DCOMPILE_TARGET=virtual
+          fi
+        displayName: Test building a sample against installed CCF (non-SGX)
+
+  - ${{ if eq(parameters.target, 'SNPCC') }}:
+      - script: |
+          set -ex
+          unsafe_build=$(grep unsafe /tmp/install_prefix) || true
+          if [[ -n "$unsafe_build" ]]; then
+            ./tests/test_install_build.sh -DUSE_UNSAFE_VERSION=ON -DCOMPILE_TARGET=virtual
+          else
+            ./tests/test_install_build.sh -DCOMPILE_TARGET=virtual
```

Comment:
What's the reason for virtual rather than snp? They're the same for now but when we diverge them it'll just work if it's snp?

Reply:
Good spot. I've fixed this now.